sklearn/inspection/tests/test_partial_dependence.py

```diff
@@ -197,7 +210,7 @@ def test_partial_dependence_helpers(est, method, target_feature):
                      [123]])
 
     if method == 'brute':
-        pdp = _partial_dependence_brute(est, grid, features, X,
+        pdp, predictions = _partial_dependence_brute(est, grid, features, X,
                                         response_method='auto')
```

Comment:
there is some PEP8 issue here.

Reply:
Fixed in https://github.com/scikit-learn/scikit-learn/pull/16619/commits/390244abfe1a87d9d9b975af9face0291d8d8c29